iOS/Flows/Conversation/Cells/MessageSequenceCollectionViewDataSource.swift

```diff
@@ -20,14 +20,16 @@ class MessageSequenceCollectionViewDataSource: CollectionViewDataSource<MessageS
     }
 
     enum ItemType: Hashable {
-        case message(cid: ConversationId,
-                     messageID: MessageId,
+        case message(messageID: MessageId,
                      showDetail: Bool = true)
-        case loadMore(cid: ConversationId)
+        case loadMore
         case placeholder
-        case initial(cid: ConversationId)
```

Comment:
These types don't need a conversation id. The message sequence controller is assigned to the data source.